.github/workflows/integration-specs.yml

```diff
@@ -0,0 +1,76 @@
+name: Integration Specs
+
+on: [push]
+
+jobs:
+  integration-specs:
+    runs-on: ubuntu-latest
+
+    services:
+      zookeeper:
+        image: confluentinc/cp-zookeeper
+        ports:
+          - 2181:2181
+
+      kafka:
+        image: confluentinc/cp-kafka
+        ports:
+          - 9092:9092
+          - 29092:29092
+        env:
+          KAFKA_BROKER_ID: 1
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+          KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:29092,PLAINTEXT_HOST://localhost:9092
+          KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+          KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
+          KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+
+    steps:
+    - uses: zendesk/checkout@v2
+    - name: Set up Ruby
+      uses: zendesk/setup-ruby@v1.58.0
+      with:
+        ruby-version: 2.7
+    - name: Build and test with RSpec
+      env:
+        RACECAR_BROKERS: localhost:9092
+      run: |
+        gem install bundler --no-document
```

Comment:
Actually the `setup-ruby` step will make sure Bundler is installed. And if you add your Gemfile.lock to the repository, it will make sure to install the same version of Bundler that you used locally.

If you add `bundler-cache: true` to the `with` part of setup-ruby, it will run `bundle install` and store the result in a cache (or fetch, if it already exists).

Read more at https://github.com/ruby/setup-ruby#usage

Reply:
Thanks! I'll try to optimize that in a follow-up PR.